config/backends.go

```diff
@@ -91,11 +91,19 @@ type Redis struct {
 	Password   string `mapstructure:"password"`
 	Db         int    `mapstructure:"db"`
 	Expiration int    `mapstructure:"expiration"`
+	Tls        RedisTLS `mapstructure:"tls"`
+}
+
+type RedisTLS struct {
+	Enabled bool `mapstructure:"enabled"`
```

Comment:
Looks like this file may need a gofmt run.

Reply:
Updated